src/docker-outside-of-docker/devcontainer-feature.json

```
@@ -1,6 +1,6 @@
 {
     "id": "docker-outside-of-docker",
-    "version": "1.5.0",
+    "version": "1.5.1",
```

```suggestion
    "version": "1.6.0",
```

Given the new additions